pkgs/development/python-modules/python-vipaccess/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, lxml
+, oath
+, pycryptodome
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "python-vipaccess";
+  version = "0.10.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1m6b7qipiaj6pz86kjhyq5m5jxxijpk58gpsdkj5bn0wjl6x1pg2";
+  };
+
+  propagatedBuildInputs = [
+    lxml
+    oath
+    pycryptodome
+    requests
+  ];
+
+  doCheck = false;
```

Comment:
Why are the tests disabled? Include a comment in the expression explaining why the tests are disabled.